build/gulp.core.js

```diff
@@ -99,6 +100,14 @@ const copyJs = () => {
   return gulp.src('src/bp/ml/svm-js/**/*.*').pipe(gulp.dest('./out/bp/ml/svm-js'))
 }
 
+const checkTranslations = cb => {
+  const reorder = process.argv.find(x => x.toLowerCase() === '--reorder')
```

Comment:
`reorder` is not used

Reply:
Good catch, I renamed fix to reorder but forgot to change it everywhere. Damn old .js files